### ISSUE TITLE
tclsh symlink missing from Tcl build from easybuild

### ISSUE THREAD

**Issue by shahzebsiddiqui** [easybuilders/easybuild-easyconfigs#5163]
I noticed that tclsh is not found when using Tcl from a easybuild. The issue is that there is no symlink which can be done easily during the build process. Otherwise this can be problematic for users trying to figure out what version to use when running the tests.

```
[siddis14@amrndhl1317 ebapps]$ ls -l "/nfs/grid/software/RHEL7/easybuild/software/MPI/intel/2017.1.132-GCC-5.4.0-2.27/impi/2017.1.132/Tcl/8.6.5/bin"
total 747
-rwxr-xr-x 1 chavom hpcswbuild  27640 Sep 12 14:35 sqlite3_analyzer
-rwxr-xr-x 1 chavom hpcswbuild 372064 Sep 12 14:34 tclsh8.6
[siddis14@amrndhl1317 ebapps]$ which tclsh
/bin/tclsh
[siddis14@amrndhl1317 ebapps]$ ls -l /bin/tclsh
lrwxrwxrwx. 1 root root 8 Mar  3  2017 /bin/tclsh -> tclsh8.5

```

**Comment by shahzebsiddiqui** [comment]
@boegel should we make the symlink change as part of all Tcl easyconfigs, that way users can use tclsh without worrying about the version number, possibly also help with code portability with different Tcl versions if users are using the EB Tcl module

**Comment by boegel** [comment]
This was fixed in #6915
